OpenDuetPkg.dsc

```diff
@@ -279,7 +296,7 @@
   MSFT:DEBUG_*_*_CC_FLAGS    = -D OC_TARGET_RELEASE=1 /FAcs -Dinline=__inline -DMDEPKG_NDEBUG /GS /kernel
   MSFT:RELEASE_*_*_CC_FLAGS  = -D OC_TARGET_RELEASE=1 /FAcs -Dinline=__inline -DMDEPKG_NDEBUG /GS /kernel
 
-  XCODE:NOOPT_*_*_CC_FLAGS   = -D OC_TARGET_RELEASE=1 -fno-unwind-tables -O0 -fstack-protector-strong -ftrivial-auto-var-init=pattern
+  XCODE:NOOPT_*_*_CC_FLAGS   = -D OC_TARGET_RELEASE=1 -fno-unwind-tables -flto -Os -fstack-protector-strong -ftrivial-auto-var-init=pattern
```

Comment:
This change transformed NOOPT into DEBUG. As now there is not difference, NOOPT is useless.

Reply:
No, NOOPT is now capable of symbolic debugging, and you can actually start it.

Reply:
The current difference with NOOPT, compared to DEBUG, is that all the debug macros are on, also non-null serial debug lib is on.

Reply:
Please see https://github.com/acidanthera/bugtracker/issues/2397 - thank you